qlib/rl/data/pickle_styled.py

```diff
@@ -233,6 +238,25 @@ def __repr__(self) -> str:
             return f"{self.__class__.__name__}({self.today}, {self.yesterday})"
 
 
+class NTIntradayProcessedData(BaseIntradayProcessedData):
```

Comment:
I think this should be put into another file, because it's not `pickle_styled`. See the headers of this file.

Reply:
I am not sure how should we re-organize related structures. If we need to move `NTIntradayProcessedData` to a separate file, where should we put `NTIntradayProcessedData` and `load_intraday_processed_data()`?